frontend/src/components/Pages/YTC/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from "react"
+import { getBaseTokensWithActiveTranches } from "../../../features/element";
+import { elementAddressesAtom } from "../../../recoil/element/atom";
+import { Token } from "../../../types/manual/types";
+import { Calculator } from "./Calculator";
+import { Ape, ApeProps } from "./Executor";
+import { useRecoilValue } from 'recoil';
+import { simulationResultsAtom } from '../../../recoil/simulationResults/atom';
+import { YTCOutput } from "../../../features/ytc/ytcHelpers";
+import { Title } from "../../Title";
+import ResultsTable from "./Table";
+import Icon from "@chakra-ui/icon";
+import { Flex } from "@chakra-ui/layout";
+
+interface YTCProps {}
+
+export const YTC: React.FC<YTCProps> = (props) => {
+
+    const [baseTokens, setBaseTokens] = useState<Token[]>([]);
+    const [resultIndex, setResultIndex] = useState<number | undefined>(undefined);
+    const simulationResults: YTCOutput[] = useRecoilValue(simulationResultsAtom);
+    const elementAddresses = useRecoilValue(elementAddressesAtom);
+
+    useEffect(() => {
+        getBaseTokensWithActiveTranches(elementAddresses).then((res) => {
+            setBaseTokens(res);
+        })
+    }, [elementAddresses])
+
+    // TODO this is a stopgap
```

Comment:
purely for my understanding - what does "stopgap" mean here?

Reply:
a temporary way of dealing with a problem or satisfying a need.

It was a bad comment though, and I added a short one explaining the function instead.